schema.graphql

```diff
@@ -522,6 +522,9 @@ type SubgraphDeployment @entity {
   activeSubgraphCount: Int!
   "Amount of Subgraph entities that were currently using this deployment when they got deprecated"
   deprecatedSubgraphCount: Int!
+
+  "The contract sources in this subgraph deployment's manifest"
+  contracts: [Contract!]!
```

Comment:
I would probably modify this to use a many-to-many relationship auxiliary entity like described [in the docs](https://thegraph.com/docs/en/developer/create-subgraph-hosted/#many-to-many-relationships).
It's unfortunately a little bit extra effort for querying, but a lot better for indexing performance (mostly db size).